client/components/logged-out-form/link-item.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import omit from 'lodash/object/omit';
+
+module.exports = React.createClass( {
+	displayName: 'LoggedOutFormLinkItem',
+
+	render: function() {
```

Comment:
could be done as es6 
so
`render() {` 

and `module.exports =` should be `export default`